internal/cmd/context/ssh_key.go

```diff
@@ -0,0 +1,211 @@
+package context
+
+import (
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+	"sync"
+
+	"github.com/spf13/cobra"
+
+	"github.com/hetznercloud/cli/internal/cmd/cmpl"
+	"github.com/hetznercloud/cli/internal/state"
+	"github.com/hetznercloud/cli/internal/state/config"
+	"github.com/hetznercloud/hcloud-go/v2/hcloud"
+)
+
+func newSSHKeyCommand(s state.State) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "ssh-key",
+		Short:                 "Manage a context's default SSH key",
+		Args:                  cobra.NoArgs,
+		TraverseChildren:      true,
+		DisableFlagsInUseLine: true,
+	}
+	cmd.AddCommand(newSSHKeyAddCommand(s))
+	cmd.AddCommand(newSSHKeyRemoveCommand(s))
+	cmd.AddCommand(newSSHKeyListCommand(s))
+	return cmd
+}
+
+func newSSHKeyAddCommand(s state.State) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "add SSH-KEY...",
+		Short:                 "Add a default SSH key to the context",
+		TraverseChildren:      true,
+		DisableFlagsInUseLine: true,
+		RunE:                  state.Wrap(s, runSSHKeyAdd),
+	}
+	cmd.Flags().String("context", "", "Name of the context to add the default SSH key(s) to")
+	_ = cmd.RegisterFlagCompletionFunc("context", cmpl.SuggestCandidates(config.ContextNames(s.Config())...))
```

Comment:
What do you think about moving this flag to `newSSHKeyCommand`, as its specified in every subcommand?

Reply:
Good idea. What do you think a good description would be?